### PR TITLE
keyboard

### PR DESCRIPTION
**Pull request by sebix**
I use these commits/changes since a while now.

- gui shortcuts: fix opening link in all occasions
- gui shortcut: messages for entering/exiting filter mode
- gui shortcuts: new mode "view" to switch views
- gui shortcuts: fix actions on aggregated news items
- gui shortcuts: show error on invalid aggregate_open command

### CODE REVIEW

File: src/gui/src/assets/keyboard_mixin.js [hunk 1]
```diff
@@ -83,8 +92,13 @@ const keyboardMixin = targetId => ({
             card.show = this.card_items[this.pos].querySelector(".card");
             card.id = this.card_items[this.pos].dataset.id;
```
Comment (b3n4kh) [inline src/gui/src/assets/keyboard_mixin.js:93]: If nothing is focused dataset can be undefined, maybe just abor the function in that case or select the first card?

image

Comment (sebix) [reply src/gui/src/assets/keyboard_mixin.js:93]: As I didn't change that line -> #78

**Comment by sebix** [comment]
@milankowww is this feature of interest for TaranisNG?

**Comment by sebix** [comment]
Thanks for merging! Great to see it is of use.

**Pull request merged**